src/Silhouette.js

```diff
@@ -140,8 +140,8 @@ class Silhouette {
      * @returns {Uint8ClampedArray} dst
      */
     colorAtLinear (vec, dst) {
```

Comment:
X and Y are still floored here, but that should be okay because we're near a pixel boundary, it'll just interpolate to the next pixel over anyway.